addons/website/models/mixins.py

```diff
@@ -133,18 +133,12 @@ def _compute_website_url(self):
     @api.multi
     def website_publish_button(self):
```

Comment:
Can't we remove this entirely ?

Reply:
Still used for Badge and Delivery. These don't have a website page.